[ERROR] {0}: Not alive and dispose was called, filename: {1}

**Describe the bug**
Running `mvn dependency-check:check` produced the following log:
```
[INFO] Error stacktraces are turned on.
[INFO] Scanning for projects...
[INFO] 
[INFO] ----------------------< com.example:project >----------------------
[INFO] Building Example Project 1.0-SNAPSHOT
[INFO]   from pom.xml
[INFO] --------------------------------[ jar ]---------------------------------
[INFO] 
[INFO] --- dependency-check:8.4.2:check (default-cli) @ project ---
[INFO] Checking for updates
[INFO] Skipping NVD check since last check was within 4 hours.
[INFO] Skipping RetireJS update since last update was within 24 hours.
[INFO] Skipping Hosted Suppressions file update since last update was within 2 hours.
[INFO] Skipping Known Exploited Vulnerabilities update check since last check was within 24 hours.
[INFO] Check for updates complete (32 ms)
[INFO] 

Dependency-Check is an open source tool performing a best effort analysis of 3rd party dependencies; false positives and false negatives may exist in the analysis performed by the tool. Use of the tool and the reporting provided constitutes acceptance for use in an AS IS condition, and there are NO warranties, implied or otherwise, with regard to the analysis or its use. Any use of the tool and the reporting provided is at the user’s risk. In no event shall the copyright holder or OWASP be held liable for any damages whatsoever arising out of or in connection with the use of this tool, the analysis performed, or the resulting report.


   About ODC: https://jeremylong.github.io/DependencyCheck/general/internals.html
   False Positives: https://jeremylong.github.io/DependencyCheck/general/suppression.html

💖 Sponsor: https://github.com/sponsors/jeremylong


[INFO] Analysis Started
[INFO] Finished Archive Analyzer (5 seconds)
[INFO] Finished File Name Analyzer (0 seconds)
[INFO] Finished Jar Analyzer (1 seconds)
[INFO] Finished Dependency Merging Analyzer (0 seconds)
[INFO] Finished Hint Analyzer (0 seconds)
[INFO] Finished Version Filter Analyzer (0 seconds)
[INFO] Created CPE Index (2 seconds)
[INFO] Finished CPE Analyzer (4 seconds)
[INFO] Finished False Positive Analyzer (0 seconds)
[INFO] Finished NVD CVE Analyzer (0 seconds)
[INFO] Finished RetireJS Analyzer (2 seconds)
[INFO] Finished Sonatype OSS Index Analyzer (0 seconds)
[INFO] Finished Vulnerability Suppression Analyzer (0 seconds)
[INFO] Finished Known Exploited Vulnerability Analyzer (0 seconds)
[INFO] Finished Dependency Bundling Analyzer (0 seconds)
[INFO] Suppression Rule had zero matches: SuppressionRule{cve={CVE-2023-44487,}}
[INFO] Finished Unused Suppression Rule Analyzer (0 seconds)
[INFO] Analysis Complete (20 seconds)
[INFO] Writing report to: C:\project\target\owasp-reports\dependency-check-report.html
[WARNING] 

One or more dependencies were identified with known vulnerabilities in Example Project:

xmlsec-2.2.4.jar (pkg:maven/org.apache.santuario/xmlsec@2.2.4, cpe:2.3:a:apache:santuario_xml_security_for_java:2.2.4:*:*:*:*:*:*:*, cpe:2.3:a:apache:xml_security_for_java:2.2.4:*:*:*:*:*:*:*) : CVE-2023-44483


See the dependency-check report for more details.


[INFO] Cache event queue destroyed: {0}
[INFO] Cache event queue destroyed: {0}
[INFO] Cache event queue destroyed: {0}
[ERROR] {0}: Not alive and dispose was called, filename: {1}
[ERROR] {0}: Not alive and dispose was called, filename: {1}
[ERROR] {0}: Not alive and dispose was called, filename: {1}
[INFO] ------------------------------------------------------------------------
[INFO] BUILD SUCCESS
[INFO] ------------------------------------------------------------------------
[INFO] Total time:  34.369 s
[INFO] Finished at: 2023-10-23T14:06:12+01:00
[INFO] ------------------------------------------------------------------------

Process finished with exit code 0

```

**Version of dependency-check used**
The problem occurs using version 8.4.2 of the maven plugin.

**Log file**
The following part of the debug log may be relevant:
```
[DEBUG] Error loading LogFactory
java.util.ServiceConfigurationError: org.apache.commons.jcs3.log.LogFactory: org.apache.commons.jcs3.log.Log4j2Factory Unable to get public no-arg constructor
    at java.util.ServiceLoader.fail (ServiceLoader.java:582)
    at java.util.ServiceLoader.getConstructor (ServiceLoader.java:673)
    at java.util.ServiceLoader$LazyClassPathLookupIterator.hasNextService (ServiceLoader.java:1233)
    at java.util.ServiceLoader$LazyClassPathLookupIterator.hasNext (ServiceLoader.java:1265)
    at java.util.ServiceLoader$2.hasNext (ServiceLoader.java:1300)
    at java.util.ServiceLoader$3.hasNext (ServiceLoader.java:1385)
    at org.apache.commons.jcs3.log.LogManager$LogFactoryHolder.createLogFactory (LogManager.java:69)
    at org.apache.commons.jcs3.log.LogManager$LogFactoryHolder.<clinit> (LogManager.java:48)
    at org.apache.commons.jcs3.log.LogManager.getLogFactory (LogManager.java:121)
    at org.apache.commons.jcs3.log.LogManager.getLog (LogManager.java:151)
    at org.apache.commons.jcs3.engine.control.CompositeCacheManager.<clinit> (CompositeCacheManager.java:80)
    at org.apache.commons.jcs3.JCS.getCacheManager (JCS.java:116)
    at org.apache.commons.jcs3.JCS.getInstance (JCS.java:159)
    at org.owasp.dependencycheck.data.cache.DataCacheFactory.getPomCache (DataCacheFactory.java:155)
    at org.owasp.dependencycheck.analyzer.CentralAnalyzer.initialize (CentralAnalyzer.java:124)
    at org.owasp.dependencycheck.Engine.lambda$loadAnalyzers$1 (Engine.java:237)
    at java.util.ArrayList.forEach (ArrayList.java:1541)
    at org.owasp.dependencycheck.Engine.loadAnalyzers (Engine.java:236)
    at org.owasp.dependencycheck.Engine.initializeEngine (Engine.java:203)
    at org.owasp.dependencycheck.Engine.<init> (Engine.java:192)
    at org.owasp.dependencycheck.Engine.<init> (Engine.java:167)
    at org.owasp.dependencycheck.Engine.<init> (Engine.java:157)
    at org.owasp.dependencycheck.maven.BaseDependencyCheckMojo.initializeEngine (BaseDependencyCheckMojo.java:2125)
    at org.owasp.dependencycheck.maven.BaseDependencyCheckMojo.runCheck (BaseDependencyCheckMojo.java:1920)
    at org.owasp.dependencycheck.maven.BaseDependencyCheckMojo.execute (BaseDependencyCheckMojo.java:1112)
    at org.apache.maven.plugin.DefaultBuildPluginManager.executeMojo (DefaultBuildPluginManager.java:126)
    at org.apache.maven.lifecycle.internal.MojoExecutor.doExecute2 (MojoExecutor.java:328)
    at org.apache.maven.lifecycle.internal.MojoExecutor.doExecute (MojoExecutor.java:316)
    at org.apache.maven.lifecycle.internal.MojoExecutor.execute (MojoExecutor.java:212)
    at org.apache.maven.lifecycle.internal.MojoExecutor.execute (MojoExecutor.java:174)
    at org.apache.maven.lifecycle.internal.MojoExecutor.access$000 (MojoExecutor.java:75)
    at org.apache.maven.lifecycle.internal.MojoExecutor$1.run (MojoExecutor.java:162)
    at org.apache.maven.plugin.DefaultMojosExecutionStrategy.execute (DefaultMojosExecutionStrategy.java:39)
    at org.apache.maven.lifecycle.internal.MojoExecutor.execute (MojoExecutor.java:159)
    at org.apache.maven.lifecycle.internal.LifecycleModuleBuilder.buildProject (LifecycleModuleBuilder.java:105)
    at org.apache.maven.lifecycle.internal.LifecycleModuleBuilder.buildProject (LifecycleModuleBuilder.java:73)
    at org.apache.maven.lifecycle.internal.builder.singlethreaded.SingleThreadedBuilder.build (SingleThreadedBuilder.java:53)
    at org.apache.maven.lifecycle.internal.LifecycleStarter.execute (LifecycleStarter.java:118)
    at org.apache.maven.DefaultMaven.doExecute (DefaultMaven.java:261)
    at org.apache.maven.DefaultMaven.doExecute (DefaultMaven.java:173)
    at org.apache.maven.DefaultMaven.execute (DefaultMaven.java:101)
    at org.apache.maven.cli.MavenCli.execute (MavenCli.java:906)
    at org.apache.maven.cli.MavenCli.doMain (MavenCli.java:283)
    at org.apache.maven.cli.MavenCli.main (MavenCli.java:206)
    at jdk.internal.reflect.NativeMethodAccessorImpl.invoke0 (Native Method)
    at jdk.internal.reflect.NativeMethodAccessorImpl.invoke (NativeMethodAccessorImpl.java:62)
    at jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke (DelegatingMethodAccessorImpl.java:43)
    at java.lang.reflect.Method.invoke (Method.java:566)
    at org.codehaus.plexus.classworlds.launcher.Launcher.launchEnhanced (Launcher.java:283)
    at org.codehaus.plexus.classworlds.launcher.Launcher.launch (Launcher.java:226)
    at org.codehaus.plexus.classworlds.launcher.Launcher.mainWithExitCode (Launcher.java:407)
    at org.codehaus.plexus.classworlds.launcher.Launcher.main (Launcher.java:348)
    at jdk.internal.reflect.NativeMethodAccessorImpl.invoke0 (Native Method)
    at jdk.internal.reflect.NativeMethodAccessorImpl.invoke (NativeMethodAccessorImpl.java:62)
    at jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke (DelegatingMethodAccessorImpl.java:43)
    at java.lang.reflect.Method.invoke (Method.java:566)
    at org.apache.maven.wrapper.BootstrapMainStarter.start (BootstrapMainStarter.java:52)
    at org.apache.maven.wrapper.WrapperExecutor.execute (WrapperExecutor.java:161)
    at org.apache.maven.wrapper.MavenWrapperMain.main (MavenWrapperMain.java:73)
Caused by: java.lang.NoClassDefFoundError: org/apache/logging/log4j/message/MessageFactory
    at java.lang.Class.getDeclaredConstructors0 (Native Method)
    at java.lang.Class.privateGetDeclaredConstructors (Class.java:3137)
    at java.lang.Class.getConstructor0 (Class.java:3342)
    at java.lang.Class.getConstructor (Class.java:2151)
    at java.util.ServiceLoader$1.run (ServiceLoader.java:660)
    at java.util.ServiceLoader$1.run (ServiceLoader.java:657)
    at java.security.AccessController.doPrivileged (Native Method)
    at java.util.ServiceLoader.getConstructor (ServiceLoader.java:668)
    at java.util.ServiceLoader$LazyClassPathLookupIterator.hasNextService (ServiceLoader.java:1233)
    at java.util.ServiceLoader$LazyClassPathLookupIterator.hasNext (ServiceLoader.java:1265)
    at java.util.ServiceLoader$2.hasNext (ServiceLoader.java:1300)
    at java.util.ServiceLoader$3.hasNext (ServiceLoader.java:1385)
    at org.apache.commons.jcs3.log.LogManager$LogFactoryHolder.createLogFactory (LogManager.java:69)
    at org.apache.commons.jcs3.log.LogManager$LogFactoryHolder.<clinit> (LogManager.java:48)
    at org.apache.commons.jcs3.log.LogManager.getLogFactory (LogManager.java:121)
    at org.apache.commons.jcs3.log.LogManager.getLog (LogManager.java:151)
    at org.apache.commons.jcs3.engine.control.CompositeCacheManager.<clinit> (CompositeCacheManager.java:80)
    at org.apache.commons.jcs3.JCS.getCacheManager (JCS.java:116)
    at org.apache.commons.jcs3.JCS.getInstance (JCS.java:159)
    at org.owasp.dependencycheck.data.cache.DataCacheFactory.getPomCache (DataCacheFactory.java:155)
    at org.owasp.dependencycheck.analyzer.CentralAnalyzer.initialize (CentralAnalyzer.java:124)
    at org.owasp.dependencycheck.Engine.lambda$loadAnalyzers$1 (Engine.java:237)
    at java.util.ArrayList.forEach (ArrayList.java:1541)
    at org.owasp.dependencycheck.Engine.loadAnalyzers (Engine.java:236)
    at org.owasp.dependencycheck.Engine.initializeEngine (Engine.java:203)
    at org.owasp.dependencycheck.Engine.<init> (Engine.java:192)
    at org.owasp.dependencycheck.Engine.<init> (Engine.java:167)
    at org.owasp.dependencycheck.Engine.<init> (Engine.java:157)
    at org.owasp.dependencycheck.maven.BaseDependencyCheckMojo.initializeEngine (BaseDependencyCheckMojo.java:2125)
    at org.owasp.dependencycheck.maven.BaseDependencyCheckMojo.runCheck (BaseDependencyCheckMojo.java:1920)
    at org.owasp.dependencycheck.maven.BaseDependencyCheckMojo.execute (BaseDependencyCheckMojo.java:1112)
    at org.apache.maven.plugin.DefaultBuildPluginManager.executeMojo (DefaultBuildPluginManager.java:126)
    at org.apache.maven.lifecycle.internal.MojoExecutor.doExecute2 (MojoExecutor.java:328)
    at org.apache.maven.lifecycle.internal.MojoExecutor.doExecute (MojoExecutor.java:316)
    at org.apache.maven.lifecycle.internal.MojoExecutor.execute (MojoExecutor.java:212)
    at org.apache.maven.lifecycle.internal.MojoExecutor.execute (MojoExecutor.java:174)
    at org.apache.maven.lifecycle.internal.MojoExecutor.access$000 (MojoExecutor.java:75)
    at org.apache.maven.lifecycle.internal.MojoExecutor$1.run (MojoExecutor.java:162)
    at org.apache.maven.plugin.DefaultMojosExecutionStrategy.execute (DefaultMojosExecutionStrategy.java:39)
    at org.apache.maven.lifecycle.internal.MojoExecutor.execute (MojoExecutor.java:159)
    at org.apache.maven.lifecycle.internal.LifecycleModuleBuilder.buildProject (LifecycleModuleBuilder.java:105)
    at org.apache.maven.lifecycle.internal.LifecycleModuleBuilder.buildProject (LifecycleModuleBuilder.java:73)
    at org.apache.maven.lifecycle.internal.builder.singlethreaded.SingleThreadedBuilder.build (SingleThreadedBuilder.java:53)
    at org.apache.maven.lifecycle.internal.LifecycleStarter.execute (LifecycleStarter.java:118)
    at org.apache.maven.DefaultMaven.doExecute (DefaultMaven.java:261)
    at org.apache.maven.DefaultMaven.doExecute (DefaultMaven.java:173)
    at org.apache.maven.DefaultMaven.execute (DefaultMaven.java:101)
    at org.apache.maven.cli.MavenCli.execute (MavenCli.java:906)
    at org.apache.maven.cli.MavenCli.doMain (MavenCli.java:283)
    at org.apache.maven.cli.MavenCli.main (MavenCli.java:206)
    at jdk.internal.reflect.NativeMethodAccessorImpl.invoke0 (Native Method)
    at jdk.internal.reflect.NativeMethodAccessorImpl.invoke (NativeMethodAccessorImpl.java:62)
    at jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke (DelegatingMethodAccessorImpl.java:43)
    at java.lang.reflect.Method.invoke (Method.java:566)
    at org.codehaus.plexus.classworlds.launcher.Launcher.launchEnhanced (Launcher.java:283)
    at org.codehaus.plexus.classworlds.launcher.Launcher.launch (Launcher.java:226)
    at org.codehaus.plexus.classworlds.launcher.Launcher.mainWithExitCode (Launcher.java:407)
    at org.codehaus.plexus.classworlds.launcher.Launcher.main (Launcher.java:348)
    at jdk.internal.reflect.NativeMethodAccessorImpl.invoke0 (Native Method)
    at jdk.internal.reflect.NativeMethodAccessorImpl.invoke (NativeMethodAccessorImpl.java:62)
    at jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke (DelegatingMethodAccessorImpl.java:43)
    at java.lang.reflect.Method.invoke (Method.java:566)
    at org.apache.maven.wrapper.BootstrapMainStarter.start (BootstrapMainStarter.java:52)
    at org.apache.maven.wrapper.WrapperExecutor.execute (WrapperExecutor.java:161)
    at org.apache.maven.wrapper.MavenWrapperMain.main (MavenWrapperMain.java:73)
Caused by: java.lang.ClassNotFoundException: org.apache.logging.log4j.message.MessageFactory
    at org.codehaus.plexus.classworlds.strategy.SelfFirstStrategy.loadClass (SelfFirstStrategy.java:50)
    at org.codehaus.plexus.classworlds.realm.ClassRealm.unsynchronizedLoadClass (ClassRealm.java:271)
    at org.codehaus.plexus.classworlds.realm.ClassRealm.loadClass (ClassRealm.java:247)
    at org.codehaus.plexus.classworlds.realm.ClassRealm.loadClass (ClassRealm.java:239)
    at java.lang.Class.getDeclaredConstructors0 (Native Method)
    at java.lang.Class.privateGetDeclaredConstructors (Class.java:3137)
    at java.lang.Class.getConstructor0 (Class.java:3342)
    at java.lang.Class.getConstructor (Class.java:2151)
    at java.util.ServiceLoader$1.run (ServiceLoader.java:660)
    at java.util.ServiceLoader$1.run (ServiceLoader.java:657)
    at java.security.AccessController.doPrivileged (Native Method)
    at java.util.ServiceLoader.getConstructor (ServiceLoader.java:668)
    at java.util.ServiceLoader$LazyClassPathLookupIterator.hasNextService (ServiceLoader.java:1233)
    at java.util.ServiceLoader$LazyClassPathLookupIterator.hasNext (ServiceLoader.java:1265)
    at java.util.ServiceLoader$2.hasNext (ServiceLoader.java:1300)
    at java.util.ServiceLoader$3.hasNext (ServiceLoader.java:1385)
    at org.apache.commons.jcs3.log.LogManager$LogFactoryHolder.createLogFactory (LogManager.java:69)
    at org.apache.commons.jcs3.log.LogManager$LogFactoryHolder.<clinit> (LogManager.java:48)
    at org.apache.commons.jcs3.log.LogManager.getLogFactory (LogManager.java:121)
    at org.apache.commons.jcs3.log.LogManager.getLog (LogManager.java:151)
    at org.apache.commons.jcs3.engine.control.CompositeCacheManager.<clinit> (CompositeCacheManager.java:80)
    at org.apache.commons.jcs3.JCS.getCacheManager (JCS.java:116)
    at org.apache.commons.jcs3.JCS.getInstance (JCS.java:159)
    at org.owasp.dependencycheck.data.cache.DataCacheFactory.getPomCache (DataCacheFactory.java:155)
    at org.owasp.dependencycheck.analyzer.CentralAnalyzer.initialize (CentralAnalyzer.java:124)
    at org.owasp.dependencycheck.Engine.lambda$loadAnalyzers$1 (Engine.java:237)
    at java.util.ArrayList.forEach (ArrayList.java:1541)
    at org.owasp.dependencycheck.Engine.loadAnalyzers (Engine.java:236)
    at org.owasp.dependencycheck.Engine.initializeEngine (Engine.java:203)
    at org.owasp.dependencycheck.Engine.<init> (Engine.java:192)
    at org.owasp.dependencycheck.Engine.<init> (Engine.java:167)
    at org.owasp.dependencycheck.Engine.<init> (Engine.java:157)
    at org.owasp.dependencycheck.maven.BaseDependencyCheckMojo.initializeEngine (BaseDependencyCheckMojo.java:2125)
    at org.owasp.dependencycheck.maven.BaseDependencyCheckMojo.runCheck (BaseDependencyCheckMojo.java:1920)
    at org.owasp.dependencycheck.maven.BaseDependencyCheckMojo.execute (BaseDependencyCheckMojo.java:1112)
    at org.apache.maven.plugin.DefaultBuildPluginManager.executeMojo (DefaultBuildPluginManager.java:126)
    at org.apache.maven.lifecycle.internal.MojoExecutor.doExecute2 (MojoExecutor.java:328)
    at org.apache.maven.lifecycle.internal.MojoExecutor.doExecute (MojoExecutor.java:316)
    at org.apache.maven.lifecycle.internal.MojoExecutor.execute (MojoExecutor.java:212)
    at org.apache.maven.lifecycle.internal.MojoExecutor.execute (MojoExecutor.java:174)
    at org.apache.maven.lifecycle.internal.MojoExecutor.access$000 (MojoExecutor.java:75)
    at org.apache.maven.lifecycle.internal.MojoExecutor$1.run (MojoExecutor.java:162)
    at org.apache.maven.plugin.DefaultMojosExecutionStrategy.execute (DefaultMojosExecutionStrategy.java:39)
    at org.apache.maven.lifecycle.internal.MojoExecutor.execute (MojoExecutor.java:159)
    at org.apache.maven.lifecycle.internal.LifecycleModuleBuilder.buildProject (LifecycleModuleBuilder.java:105)
    at org.apache.maven.lifecycle.internal.LifecycleModuleBuilder.buildProject (LifecycleModuleBuilder.java:73)
    at org.apache.maven.lifecycle.internal.builder.singlethreaded.SingleThreadedBuilder.build (SingleThreadedBuilder.java:53)
    at org.apache.maven.lifecycle.internal.LifecycleStarter.execute (LifecycleStarter.java:118)
    at org.apache.maven.DefaultMaven.doExecute (DefaultMaven.java:261)
    at org.apache.maven.DefaultMaven.doExecute (DefaultMaven.java:173)
    at org.apache.maven.DefaultMaven.execute (DefaultMaven.java:101)
    at org.apache.maven.cli.MavenCli.execute (MavenCli.java:906)
    at org.apache.maven.cli.MavenCli.doMain (MavenCli.java:283)
    at org.apache.maven.cli.MavenCli.main (MavenCli.java:206)
    at jdk.internal.reflect.NativeMethodAccessorImpl.invoke0 (Native Method)
    at jdk.internal.reflect.NativeMethodAccessorImpl.invoke (NativeMethodAccessorImpl.java:62)
    at jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke (DelegatingMethodAccessorImpl.java:43)
    at java.lang.reflect.Method.invoke (Method.java:566)
    at org.codehaus.plexus.classworlds.launcher.Launcher.launchEnhanced (Launcher.java:283)
    at org.codehaus.plexus.classworlds.launcher.Launcher.launch (Launcher.java:226)
    at org.codehaus.plexus.classworlds.launcher.Launcher.mainWithExitCode (Launcher.java:407)
    at org.codehaus.plexus.classworlds.launcher.Launcher.main (Launcher.java:348)
    at jdk.internal.reflect.NativeMethodAccessorImpl.invoke0 (Native Method)
    at jdk.internal.reflect.NativeMethodAccessorImpl.invoke (NativeMethodAccessorImpl.java:62)
    at jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke (DelegatingMethodAccessorImpl.java:43)
    at java.lang.reflect.Method.invoke (Method.java:566)
    at org.apache.maven.wrapper.BootstrapMainStarter.start (BootstrapMainStarter.java:52)
    at org.apache.maven.wrapper.WrapperExecutor.execute (WrapperExecutor.java:161)
    at org.apache.maven.wrapper.MavenWrapperMain.main (MavenWrapperMain.java:73)
[DEBUG] Found LogFactory for slf4j
[DEBUG] Setting property [{0}] to [{1}].
[DEBUG] Setting property [{0}] to [{1}].
[DEBUG] Setting property [{0}] to [{1}].
[DEBUG] Setting property [{0}] to [{1}].
[DEBUG] Setting property [{0}] to [{1}].
[DEBUG] Setting property [{0}] to [{1}].
[DEBUG] Setting property [{0}] to [{1}].
[DEBUG] Setting property [{0}] to [{1}].
[DEBUG] Setting property [{0}] to [{1}].
[DEBUG] Setting property [{0}] to [{1}].
[DEBUG] Setting property [{0}] to [{1}].
[DEBUG] Setting property [{0}] to [{1}].
[DEBUG] Setting property [{0}] to [{1}].
[DEBUG] Setting property [{0}] to [{1}].
[DEBUG] Setting property [{0}] to [{1}].
[DEBUG] Setting property [{0}] to [{1}].
[DEBUG] Setting property [{0}] to [{1}].
[DEBUG] Setting property [{0}] to [{1}].
[DEBUG] Setting property [{0}] to [{1}].
[DEBUG] Setting property [{0}] to [{1}].
[DEBUG] Setting property [{0}] to [{1}].
[DEBUG] Setting property [{0}] to [{1}].
[DEBUG] Setting property [{0}] to [{1}].
[DEBUG] Setting property [{0}] to [{1}].
[DEBUG] Setting property [{0}] to [{1}].
[DEBUG] Creating DiskCache for attributes = {0}
[DEBUG] threadPoolName = [{0}] poolType = {1}
[DEBUG] Constructed: {0}
```

Certainly it appears the logging has not initialised correctly.

**Expected behavior**
No errors logged

**Additional context**
Upgraded from 8.4.0.

--- a/pom.xml
+++ b/pom.xml
@@ -1040,7 +1040,7 @@ Copyright (c) 2012 - Jeremy Long
             <dependency>
                 <groupId>io.github.jeremylong</groupId>
                 <artifactId>jcs3-slf4j</artifactId>
-                <version>1.0.0</version>
+                <version>1.0.1</version>
             </dependency>
             <dependency>
                 <groupId>commons-validator</groupId>

